### PR TITLE
src: html: Fix alerts not showing

### PR DESCRIPTION
**Pull request by joaoantoniocardoso**
Helps #79

### CODE REVIEW

File: src/html/index.html [hunk 1]
```diff
@@ -162,7 +162,7 @@ <h3>Name: {{ stream.video_and_stream.name }}</h3>
                 },
                 checkResponse: async function(response) {
                     if (!response.ok) {
-                        const text = await response.text().replaceAll("\\n", "\n").replaceAll("\\\"", "\"")
+                        const text = await response.text().then((text) => text.replace(/^"(.+(?="$))"$/, '$1').replaceAll("\\n", "\n").replaceAll("\\\"", "\""))
```
Comment (patrickelectric) [inline src/html/index.html:165]: Can you be a bit more clear about what this regex is replacing ?

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: src/html/index.html (hunk 1 above)
Comment (joaoantoniocardoso) [reply src/html/index.html:165]: In sequence:
1. get content from inside the external double quotes pair
2. unescape new lines
3. unescape double quotes

Comment (patrickelectric) [reply src/html/index.html:165]: Ok, I would recommend to add a comment for the first regex and a minimal example of what it should match and what is the content of text just for the clarity. A look head/behind regex is never trivial.

Comment (joaoantoniocardoso) [reply src/html/index.html:165]: that's fair

Comment (joaoantoniocardoso) [reply src/html/index.html:169]: @patrickelectric done

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**